contrib/Dockerfile

```diff
@@ -11,53 +13,66 @@ RUN apt-get -qq update -y \
                clang-tidy \
                cmake \
                flex \
+               g++ \
                gawk \
+               gcc \
                git \
-               golang \
+               gperf \
                graphviz \
-               iverilog \
+               help2man \
                less \
                libboost-dev \
                libboost-filesystem-dev \
                libboost-iostreams-dev \
                libboost-program-options-dev \
+               libboost-python-dev \
+               libboost-system-dev \
                libboost-thread-dev \
                libeigen3-dev \
                libffi-dev \
+               libfl-dev \
                libftdi-dev \
-               libglib2.0-dev \
+               libpython3-dev \
                libusb-1.0-0-dev \
-               libpixman-1-dev \
                libreadline-dev \
                lld \
                llvm \
-               mercurial \
+               make \
                nano \
-               ninja-build \
+               ncdu \
+               perl \
                pkg-config \
+               psmisc \
+               python-is-python3 \
                python3 \
+               python3-full \
                python3-dev \
+               python3-pip \
+               python3-pytest \
+               sdcc \
                tcl-dev \
-               verilator \
                vim \
                xdot \
-               sdcc \
-               splint \
-               cmake \
-               gcc-arm-none-eabi \
-               libnewlib-arm-none-eabi \
-               libstdc++-arm-none-eabi-newlib \
+               zlib1g-dev \
     && rm -rf /var/lib/apt/lists/*
 
+# Enable bash completion
+RUN sed -i '/#if ! shopt -oq posix; then/ s/^#//' /etc/bash.bashrc
```

Comment:
Since all the bash completion stuff is already in the /etc/bash.bashrc file I figured the most clear solution to enable completion would be to use the what is already there.